users/init.sls

```diff
@@ -29,13 +35,22 @@ group_{{ name }}:
 {{ set_p('members', g)|indent(4, True) }}
 {% endfor %}
 
+{% for id, u in absent_users|dictsort %}
```

Comment:
instead of having a separate loop you could add a pillar *paramter* that changes from ``present`` to ``absent`` in the other loop(s)